searx/templates/oscar/stats.html

```diff
@@ -27,15 +27,15 @@ <h1>{{ _('Engine stats') }}{% if selected_engine_name %} - {{ selected_engine_na
                     <table class="table table-hover table-condensed table-striped engine-stats">
                         <tr>
                             <th scope="col" class="engine-name">{{ th_sort('name', _("Engine name")) }}</th>
-                            <th scope="col" class="engine-scores">{{ th_sort('score', _('Scores')) }}</th>
+                            <th scope="col" class="engine-score">{{ th_sort('score', _('Scores')) }}</th>
                             <th scope="col" class="result-count">{{ th_sort('result_count', _('Result count')) }}</th>
                             <th scope="col" class="response-time">{{ th_sort('time', _('Response time')) }}</th>
                             <th scope="col" class="engine-reliability">{{ th_sort('reliability', _('Reliability')) }}</th>
                         </tr>
                         {% for engine_stat in engine_stats.get('time', []) %}
                         <tr>
                             <td class="engine-name"><a href="{{ url_for('stats', engine=engine_stat.name|e) }}">{{ engine_stat.name }}</a></td>
-                            <td class="engine-scores">
+                            <td class="engine-score">
```

Comment:
`engine-score` -> `engine-scores` same in the .less files above.
It exists before this PR.